Formula/uftp.rb

```diff
@@ -0,0 +1,53 @@
+class Uftp < Formula
+  desc "secure, reliable, efficient multicast file transfer program"
+  homepage "http://uftp-multicast.sourceforge.net/"
+  url "https://downloads.sourceforge.net/project/uftp-multicast/source-tar/uftp-4.9.3.tar.gz"
+  sha256 "9e9215af0315257c6cc4f40fbc6161057e861be1fff10a38a5564f699e99c78f"
+
+  depends_on "openssl"
+
+  def install
+    system "make", "OPENSSL=#{Formula["openssl"].opt_prefix}", "DESTDIR=#{prefix}", "install"
+    # the makefile installs into DESTDIR/usr/..., move everything up one and remove usr
```

Comment:
Happy to merge anyway but it's probably worth requesting they use `PREFIX` rather than `DESTDIR`; all distributions would be able to use it with that variable and it's generally the standard.